ccmbioinfo/lfs
# CARDS-1736: Add ACL restriction for clinics

[Jira link](https://phenotips.atlassian.net/browse/CARDS-1736)

This PR adds an ACL restriction that only applies if the user belongs to a group specified by the Visit Information form.

**To test**
1. Create a new clinic
![image](https://user-images.githubusercontent.com/4656440/172628106-6353498a-1abc-49a9-a29f-be9d37d16906.png)
2. Find the clinic in Composum under /Proms/ClinicMapping
![image](https://user-images.githubusercontent.com/4656440/172640921-359fcebb-a9b3-4679-983b-9307b60cb462.png)
3. Create a new Visit Information form, fill out the Clinic question with `/Proms/ClinicMapping/<id>` and save
![image](https://user-images.githubusercontent.com/4656440/172641166-f38af6b9-4247-4e5b-bd8b-42b10cac37f4.png)
4. Under Administration->User->Create a new test user
![image](https://user-images.githubusercontent.com/4656440/172641372-fcbd3f57-a155-41b8-b5fe-5a674f6ed14c.png)
5. Under Administration->Groups, add the new user to both TrustedUser and the clinic group generated in step 1
![image](https://user-images.githubusercontent.com/4656440/172641600-4eb9bd36-4c95-4966-bd51-72fe3aa265db.png)
6. Verify that you can see the Visit Information form from step 3 when signed in as your new test user
7. If you remove them from the clinic group in step 5, they will no longer be able to see the forms.
8. Visit Information forms without the cards:clinic set to `/Proms/ClinicMapping/<id>` should still be viewable.
